var/spack/repos/builtin/packages/py-tensorboard-data-server/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import glob
+
+from spack import *
+
+
+class PyTensorboardDataServer(PythonPackage):
+    """Fast data loading for TensorBoard"""
+
+    homepage = "https://github.com/tensorflow/tensorboard/tree/master/tensorboard/data/server"
+    git      = "https://github.com/tensorflow/tensorboard"
+
+    version('0.6.1', commit='6acf0be88b5727e546dd64a8b9b12d790601d561')
+
+    depends_on('python@3.6:', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+    depends_on('rust', type='build')
+
+    def setup_build_environment(self, env):
+        env.set('CARGO_HOME', self.stage.source_path)
+
+    def install(self, spec, prefix):
```

Comment:
If we do:
```suggestion
    @run_before('install')
    def build(self):
```
Then we can remove the pip/wheel stuff below. This is optional, just a suggestion.

Reply:
This package does not install correctly with the install method removed. I think it has to do with building with cargo.